lib/browser/init.ts

```diff
@@ -15,6 +16,8 @@ require('../common/reset-search-paths')
 // Import common settings.
 require('@electron/internal/common/init')
 
+process.electronBinding('event_emitter').setEventEmitterPrototype(EventEmitter.prototype)
```

Comment:
For sanity / safety this should probably be called before `reset-search-paths` as that's what prepares the `electron` module that you "require"

Reply:
Happy to do so but I'm not quite sure why that would make a difference here? Can any user code run between `reset-search-paths` and this line?